### PR TITLE
feat(site): add numbers to SSHButton content for clarity

### PR DESCRIPTION
**Pull request by benwinding**
## Summary
So when I first used this, I stupidly didn't realize that you needed to do both steps. This PR adds numbers to the commands to make that a little clearer 🙏 

## Screenshots
| Before | After |
| - | - |
| - | _Note numbers next to steps_ |
| image | image |

### CODE REVIEW

File: site/src/modules/resources/SSHButton/SSHButton.tsx [hunk 1]
```diff
@@ -92,6 +82,15 @@ export const SSHButton: FC<SSHButtonProps> = ({
...
 };
 
+const SSHStep = (props: { helpText: string; codeExample: string }) => (
+	<li style={{ listStylePosition: "inside" }}>
+		<HelpTooltipText style={{ display: "inline" }}>
+			<strong css={styles.codeExampleLabel}>Connect to the agent:</strong>
+		</HelpTooltipText>
+		<CodeExample secret={false} code={props.codeExample} />
+	</li>
+);
```
Comment (benwinding) [inline site/src/modules/resources/SSHButton/SSHButton.tsx:92]: Thanks for reviewing @aslilac 🙏 

Changed to use semantic HTML `<li>` tags _(did it naively to reduce changes, but agree semantic html is better)_

image

**Comment by dannykopping** [comment]
recheck

File: site/src/modules/resources/SSHButton/SSHButton.tsx [hunk 2]
```diff
@@ -92,6 +82,15 @@ export const SSHButton: FC<SSHButtonProps> = ({
 	);
 };
 
+const SSHStep = (props: { helpText: string; codeExample: string }) => (
+	<li style={{ listStylePosition: "inside" }}>
+		<HelpTooltipText style={{ display: "inline" }}>
+			<strong css={styles.codeExampleLabel}>Connect to the agent:</strong>
```
Comment (aslilac) [inline site/src/modules/resources/SSHButton/SSHButton.tsx:88]: I think you meant to use the `helpText` prop here

**Comment by benwinding** [comment]
Hi @dannykopping 👋 

> @benwinding did you sign the CLA?

Sorry just getting permission from my company to sign the CLA, will comment an update when it's signed 🙏

**Review by aslilac — COMMENTED** [review]
Typically we declare the props type as a separate statement, use the `FC` type, and use argument destructuring to get the props values, like so:

```tsx
export interface SSHButtonProps {
	workspaceName: string;
	agentName: string;
	sshPrefix?: string;
}

export const SSHButton: FC<SSHButtonProps> = ({
	workspaceName,
	agentName,
	sshPrefix,
}) => {
```

`SSHStep` should do the same.

**Review by aslilac — COMMENTED** [review]
@matifali I already reviewed this, I'm waiting for the last review to be addressed.

File: site/src/modules/resources/SSHButton/SSHButton.tsx (hunk 2 above)
Comment (benwinding) [reply site/src/modules/resources/SSHButton/SSHButton.tsx]: Oh jeeze, you're right 🤦‍♂️ 

Fixed in abf9abd58cf4ca7fdb54acebb5f5c45087ca4ab7

**Review by aslilac — COMMENTED** [review]
> Typically we declare the props type as a separate statement, use the `FC` type, and use argument destructuring to get the props values, like so:

Still needs this addressed, but otherwise looks good!

**Review by aslilac — APPROVED** [review]

**Comment by aslilac** [comment]
@dannykopping @deansheather this pr is ready to go, we're just waiting on the cla. can one of you help with that since the bot isn't working?

**Comment by deansheather** [comment]
Hi @benwinding could you please comment the following once you've read and agree to the [CLA document](https://github.com/coder/cla/blob/main/README.md)?
```
I have read the CLA Document and I hereby sign the CLA
```

We've have some issues with the CLA bot over the last week or so, so I apologize that we have to do it the manual way

**Comment by benwinding** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by deansheather** [comment]
recheck

**Comment by deansheather** [comment]
Thanks for your contribution! Sorry about the difficulties merging due to the CLA bot issues 🙇

**Pull request merged**